I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 41, in script
    $ repeated_string = "Hello " * count
  File "game/script.rpy", line 41, in <module>
    $ repeated_string = "Hello " * count
NameError: name 'count' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 41, in script
    $ repeated_string = "Hello " * count
  File "C:\Users\juliu\Downloads\renpy-8.2.3-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\juliu\Downloads\renpy-8.2.3-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 41, in <module>
    $ repeated_string = "Hello " * count
NameError: name 'count' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
test 1.0
Fri Jun 28 15:41:49 2024
